cmd/mt-whisper-importer-reader/main.go

```diff
@@ -173,11 +188,18 @@ func processFromChan(files chan string, wg *sync.WaitGroup) {
 		req.Header.Set("Content-Type", "application/json")
 		req.Header.Set("Content-Encoding", "gzip")
 
-		_, err = client.Do(req)
+		if len(*httpAuth) > 0 {
+			req.Header.Add("Authorization", "Basic "+base64.StdEncoding.EncodeToString([]byte(*httpAuth)))
```

Comment:
Rather than manually adding the header, you can call
```
req.URL.User = url.UserPassword(user, pass)
```
But as you have the user/pass in a single string, what you are doing is probably easier.